docs/getting-started-in-5-minutes.md

```diff
@@ -10,148 +10,136 @@ This guide provides a general overview of running smart contracts on the Gear ne
 
 For this example, a demo environment that emulates the real Gear decentralized network will be used.
 
-
 ## 1. Prerequisites
 
 1. For your convenience, it is recommended that you create a dedicated directory for everything Gear-related. The rest of the article will assume that you are using the paths suggested. To create a folder in your home directory and navigate to it, type:
 
-    ```bash
-    mkdir -p gear
-    cd ~/gear
-    ```
+   ```bash
+   mkdir -p gear
+   cd ~/gear
+   ```
```

Comment:
Why 3 spaces?
```suggestion
    ```bash
    mkdir -p gear
    cd ~/gear
    ```
```

Reply:
Not my article, but I have removed all unnecessary spaces in my opinion